### PR TITLE
Lots of changes! New scenario! New portrait! Bugs fixed!

### LINKED ISSUE #112: Name to basic healing skill learned on S14b
Suggestion:

14b_Bontom line 33
```
id=basic_healing
name= _ "basic healing"
```

### PR DESCRIPTION
**Pull request by cooljeanius**
- add @Toranks to the credits
- new portrait for Kyobaine's winter clothing, contributed by OneHourLate, a.k.a. Mells or ohlyf (from Discord, who I also commissioned my last few portraits from). The handling for this one was a bit more complicated, so hopefully I haven't made issue #16 too much worse...
- I made an attempt at improving the issue #19 situation, but it didn't exactly work out to my liking, so it's hidden behind debug mode for now.
- New debug mode contextual menu option for displaying messages from arbitrary units, for the purposes of testing issue #16 
- Add a macro that I thought I'd included in #109 (and even used there), but looks like I actually forgot... (it's for giving experience to units)
- New scenario! This pull request includes S15c, "Raedwood East", for issue #72. It's still not fully completed to my satisfaction, so it's currently only accessible via debug mode, but it's at least in a place where I think I can at least submit it.
- Mark more units as unrenamable for issue #114 
- Adjust recruitment patterns for issue #116 
- fix #112 
- Something for issue #7 that I thought I'd already submitted in a previous PR (but, actually hadn't)

### CODE REVIEW

File: scenarios/25_The_Awakening.cfg [hunk 1]
```diff
@@ -286,13 +287,15 @@
...
         facing=ne
+        # Spearman = fighter, Javelineer = fighter, Bowman = archer, Peasant = fighter, Swordsman = fighter, Mage = mixed fighter
         recruit=Spearman,Javelineer,Bowman,Peasant,Swordsman,Mage
         {INCOME4 9 7 5 3}
         {GOLD4 250 220 190 160}
         [ai]
             {NO_SCOUTS}
             passive_leader=no
-            recruitment_pattern=fighter,fighter,healer,fighter
+            # FIXME: what to do about the "healer" entry here?
```
Comment (cooljeanius) [inline scenarios/25_The_Awakening.cfg:297]: @nemaara so, do you think we should:
1. Give Huon access to White Mages, to keep his faction entirely Loyalist
2. Give Huon access to Dune Herbalists, which, as Level 1 units, shouldn't disrupt the balance as much as White Mages might
3. Just remove the "healer" entry from this recruitment pattern
4. Just leave everything as is here

?

Comment (nemaara) [reply scenarios/25_The_Awakening.cfg:297]: White Mage is fine.

**Review by nemaara — APPROVED** [review]
I don't have time to playtest the new scenario so I'm gonna assume it's good LOL.

File: scenarios/25_The_Awakening.cfg (hunk 1 above)
Comment (cooljeanius) [reply scenarios/25_The_Awakening.cfg]: OK, White Mage recruits allowed in [2c0ea94](https://github.com/nemaara/A_New_Order/pull/117/commits/2c0ea94256fc1d7a8a9d5c3bc00f751e2ab0500c)

**Comment by cooljeanius** [comment]
> I don't have time to playtest the new scenario so I'm gonna assume it's good LOL.

So, besides the new scenario, any feedback on the new portrait, or the dialogue changes to the existing scenarios?

File: macros/elvish_macros.cfg [hunk 2]
```diff
@@ -159,22 +191,70 @@
...
+                {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"I couldn't agree more. Call me when you have new shoes."}
+            [/else]
+        [/if]
 
         {MSGOPTI2 (Gawen Hagarthen) (portraits/gawen_winter.png) (_"Well...")
         (_"All right, how about I give you 25 gold pieces instead of 20?") (yes)
         (_"All right, we will wait for the spring.") (no)}
         {IF ano_opt equals yes}
         {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"You think you can bribe me, an elvish druid, with gold? Oh well. This will allow me to buy better winter clothes."}
-        # TODO: come up with new art to show Kyobaine's better winter clothes
```
Comment (cooljeanius) [inline macros/elvish_macros.cfg:220]: So, I kind of wanted to go even further with the banter here, but decided to hold myself back... @nemaara WDYT about me possibly including additional banter here in a future PR?

**Comment by cooljeanius** [comment]
> ### maps/RaedwoodEast.map
> 
> 

...hey what happened to this image?

File: macros/elvish_macros.cfg (hunk 2 above)
Comment (nemaara) [reply macros/elvish_macros.cfg:220]: Go for it. Just don't add delays so people can hit esc if they want.

**Comment by nemaara** [comment]
Nah the other changes look good. Feel free to merge whenever.

**Comment by macabeus** [comment]
@cooljeanius 
> @macabeus did your map diff thing break again?

Probably it's broken again 😞 
I would need to spend time fixing it but I'm using all my spare time [on another personal project](https://github.com/gambiconf/gambiconf.github.io)

I would be glad if you can take a look at it.
Feel free to send me a message on Discord (`trickster#6487`) or mention me in an issue/pull request when debugging it.

**Pull request merged**